service/java/com/android/server/wifi/WifiConfigManager.java

```diff
@@ -1606,8 +1611,8 @@ public boolean removePasspointConfiguredNetwork(@NonNull String configKey) {
     public boolean isInFlakyRandomizationSsidHotlist(int networkId) {
         WifiConfiguration config = getConfiguredNetwork(networkId);
         return config != null
-                && config.macRandomizationSetting == WifiConfiguration.RANDOMIZATION_PERSISTENT
-                && mDeviceConfigFacade.getRandomizationFlakySsidHotlist().contains(config.SSID);
+                && config.macRandomizationSetting != WifiConfiguration.RANDOMIZATION_NONE
+                && mRandomizationFlakySsidHotlist.contains(config.SSID);
```

Comment:
getting a build error:

```
frameworks/opt/net/wifi/service/java/com/android/server/wifi/WifiConfigManager.java:1615: error: cannot find symbol
                && mRandomizationFlakySsidHotlist.contains(config.SSID);
                   ^
  symbol:   variable mRandomizationFlakySsidHotlist
  location: class WifiConfigManager
```

Reply:
Sorry, can you try the latest commit.
I fixed this but forgot to push.